docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
-       - The North service could crash if it retrieved invalid JSON while processing a reconfiguration request. This was addressed by adding an exception handler to prevent the crash.
+       - The documentation for the control notification plugin has been updated to include examples for all destinations of control requests.
        - Windowed averages in the notification service preserve the type of the input data when creating the averages. This does not work well for integer values and has been changed such that integer values are promoted to floating point when using windowed averages for notification rule input.
        - The notification service documentation has been updated to include examples of notifications based on statistics and audit logs.
        - The notification mechanism has been updated to accept raw statistics and statistics rates as an input for notification rules. This allows alerts to be raised for pipeline flows and other internal tasks that generate statistics.
-       - The control dispatcher now has access to the audit logging system.
        - Notifications can now register for audit log entries to be sent to notification rules. This allows notification to be made based on internal state changes of the system.
+       - The north opcuaclient has been updated to support multiple values in a single write.
+       - The north opcuaclient plugin has been updated to support security mode and security policy.
+       - The fledge-north-kafka plugin has been updated to allow for username and password authentication to be supplied when connecting to the Kafka server.
+       - Compression functionality has been added to the fledge-north-kafka.
+       - The average and watchdog rules have been updated to allow selection of data sources other than the readings to be sent to the rules.
+       - The email notification plugin has been updated to allow custom alert messages to be created.
+       - The email notification delivery plugin has been updated to hide the password from view.
+       - Some devices were not compatible with the optimized block reading of registers performed by the ModbusC south plugin. The ModbusC plugin has been updated to provide controls that can determine how it reads data from the modus device. This allows single register reads, single object reads and the current optimized block reads.
+       - The S2OPCUA South plugin now supports an optional datapoint in its Readings that shows the full path of the OPC UA Variable in the server's namespace.
+       - The option to configure and use a username and password for authentication to the MQTT broker has been added to the fledge-south-mqtt plugin.
+       - A set of Jiras has been created for the S2OPCUA South plugin to fully incorporate all unique features of the FreeOPCUA South plugin.
```

Comment:
I don't think this item should be in the public release notes document at all. This was just for us to itemize how much would be needed to allow S2OPCUA to be our only OPC UA south plugin.